src/js/controller/DrawingController.js

```diff
@@ -252,6 +267,12 @@
   ns.DrawingController.prototype.onMouseup_ = function (event) {
     var frame = this.piskelController.getCurrentFrame();
     var coords = this.getSpriteCoordinates(event.clientX, event.clientY);
+    if (typeof event.changedTouches != 'undefined' &&
+        typeof event.changedTouches[0] != 'undefined' &&
+        typeof event.changedTouches[0].pageX != 'undefined' &&
+        typeof event.changedTouches[0].pageY != 'undefined') {
```

Comment:
Maybe we could use changedTouches as well in the other two event handlers ? 

From reading the spec and quick testing I think it should give us the relevant information for the touchstart and touchmove events as well.Then we could mutualize the check done in those if in a helper function (hasChangedTouchPoint(event) ?)